ffs/types.go

```diff
@@ -56,8 +56,10 @@ func (i APIID) String() string {
 type JobStatus int
 
 const (
+	// Unspecified indicates a default or empty value
+	Unspecified JobStatus = iota
```

Comment:
This feels right to me, but let me know if it doesn't to you. The idea is this would be the default value. It's never used otherwise.

Reply:
I thought about this when I moved to starting at 1. I preferred to avoid giving name to something that doesn't have a meaning about "Job Status". But no strong opinion since I had my doubts at that moment too. LGTM.